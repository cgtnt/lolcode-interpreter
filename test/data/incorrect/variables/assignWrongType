HAI 

I HAS A VARONE ITZ 10
I HAS A VARTWO ITZ A YARN
VARTWO R VARONE

VISIBLE VARTWO 

KTHXBYE
