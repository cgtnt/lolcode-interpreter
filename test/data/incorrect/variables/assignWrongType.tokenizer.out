@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_DECLARE_VAR:I HAS A-@3:T_T_IDENTIFIER:VARONE-@3:T_DECLARE_SET_VAR:ITZ-@3:T_T_INT:10-@4:T_COMMAND_TERMINATOR:
-@4:T_DECLARE_VAR:I HAS A-@4:T_T_IDENTIFIER:VARTWO-@4:T_DECLARE_TYPE_VAR:ITZ A-@4:T_TI_STRING:YARN-@5:T_COMMAND_TERMINATOR:
-@5:T_T_IDENTIFIER:VARTWO-@5:T_ASSIGN:R-@5:T_T_IDENTIFIER:VARONE-@7:T_COMMAND_TERMINATOR:
-@7:T_WRITE_STDOUT:VISIBLE-@7:T_T_IDENTIFIER:VARTWO-@9:T_COMMAND_TERMINATOR:
-@9:T_END:KTHXBYE-@10:T_COMMAND_TERMINATOR:
-@10:T_EOF:
